src/viewer.ts

```diff
@@ -500,12 +506,11 @@ class Viewer {
             //     name: 'heli_equirect',
             //     width: 2048,
             //     height: 1024,
-            //     type: pc.TEXTURETYPE_RGBM
+            //     type: pc.TEXTURETYPE_RGBM.toString()
```

Comment:
Why is `toString` needed here?

Reply:
This was needed because typescript and incorrect docs, fixed in https://github.com/playcanvas/engine/pull/2511/files